### ISSUE TITLE
Simkl List - metadata 

### ISSUE THREAD

**Issue by guenguer** [Goldenfreddy0703/Otaku#239]
Somehow clearart and metadata isn't showing if I'm using my simkl watchlist.

It works with anilist and MAL but I prefer simkl as an all in one solution.

Is there a way to fix this?

**Comment by guenguer** [comment]
Sure. Here are some Screenshots.
1. Widget of my currently watching simkl list
2. widget - coming up next season
3. browsing my currently watching list
4. Episode list in simkl 

![simkl widget](https://github.com/Goldenfreddy0703/Otaku/assets/144166470/b3b0d07a-f6a7-4fba-a550-f0a933330c43)
currently watching 

![Screenshot_20240415-102646](https://github.com/Goldenfreddy0703/Otaku/assets/144166470/294bd2a4-7cff-4360-9e11-f1ddfc3c3161)

![Screenshot_20240415-103203](https://github.com/Goldenfreddy0703/Otaku/assets/144166470/1cee41e5-9c14-4f62-b5b2-fbc53d73e94d)

![Screenshot_20240415-102751](https://github.com/Goldenfreddy0703/Otaku/assets/144166470/6e97b7a7-9007-455a-b70d-b741ac65d7e6)

**Comment by Goldenfreddy0703** [comment]
Heyyyy so i was able to replicate the issue on Fentastic but i also tried replicating the issue on Arctic Horizon 2 and Arctic Fuse and here are my results. 

Fentastic: Description and stuff did not show fine for me for some odd reason.
![image](https://github.com/Goldenfreddy0703/Otaku/assets/62456796/99afd3d9-595e-44bb-819a-4e549af7e2ba)

Arctic Fuse: Description and stuff showed fine for me.
![image](https://github.com/Goldenfreddy0703/Otaku/assets/62456796/94cb932f-b3ad-44ed-9649-16046b1f2254)

Arctic Horizon 2: Description and stuff showed fine for me.
![image](https://github.com/Goldenfreddy0703/Otaku/assets/62456796/99df8d62-f1a4-4a69-8a11-4ead551e3960)

Hey @ivarbrandt, im very sorry for pinging or bothering you but i do not have much knowledge on Widgets and im not sure whats going on here, would you happen to know why a few of his widgets may not be showing a description? When testing Arctic Fuse and Arctic Horizon 2, they did fine but Fentastic was the only skin that had problems with some of these Widgets and im not sure why.

If you could give me a reply on this, that would be Fentastic!!!
Thank you

**Comment by guenguer** [comment]
I love fentastic tho :)
So it's a skin issue? 

I mean it's not a huge issue, I just wondered why that is.

**Comment by guenguer** [comment]
It's only in simkl tho. Not in anilist or MAL. Or anywhere else. Maybe this info helps.

**Comment by Goldenfreddy0703** [comment]
Yeah I think that's odd and I'm not too sure why cause Arctic Fuse and Horizon works but not fantastic, today I can see if I can look I to simkl watchlist and see if I can code it in an alternative way maybe.

**Comment by ivarbrandt** [comment]
The fact that the problem only appears in SIMKL lists make me think that it's an addon issue. But the fact that it works properly on other skins makes me think the issue lies in FENtastic.

I'm stumped. It doesn't make any sense.

**Comment by Goldenfreddy0703** [comment]
> The fact that the problem only appears in SIMKL lists make me think that it's an addon issue. But the fact that it works properly on other skins makes me think the issue lies in FENtastic.
> 
> I'm stumped. It doesn't make any sense.

Hey @ivarbrandt, thank you very much for replying, im stumped as well but will try to look into this tonight sense i have time tonight. I mean it could just be how Simkl.py in WatchlistFlavor is setup but again, not too sure but will look into this tonight.
Again, thank you for replying!!!!

**Comment by Goldenfreddy0703** [comment]
Hey so about the Simkl.py, im unfortunately not sure whats causing it, When Simkl watchlist was created, it was originally created by @joezito129 and he a did a fantastic job adding simkl watchlist but he could be on his hiatus at the moment so im not sure when he will come back but hopefully either @ivarbrandt may have a fix for his skin or @joezito129 can probably come back and look into this more. If anybody knows a fix for this, please feel free to reply. We will put this issue on hold until someone has a fix very soon. 

Thank you

**Comment by guenguer** [comment]
That's ok, I'm happy that simkl is even available. Underrated service :)

@Goldenfreddy0703 best anime addon available
@ivarbrandt love FENtastic

**Comment by joezito129** [comment]
When doing a request from Simkl api, they dont provide some of the metadata like anilist and myanimelist so I never added it.

**Comment by Goldenfreddy0703** [comment]
Honestly I think I'm going crazy now cause I tested today and I swear, I was getting description and all and idk what happened so I'm going to contact simkl and see if they provide them in watchlist.

**Comment by theasguard** [comment]
i use an estuary mod skin and can confirm simkl doesn't display info in those sections on it either, on the other hand MAL, Anilist, and i believe kitsu all display info along with on the episodes within those sections

**Comment by Goldenfreddy0703** [comment]
Ok i have come to the conclusion that this is most likely a Simkl issue or a lack of information on Otaku Simkl Watchlist, I don't understand how or why even tho i tested it and im confused as everyone else so im going to be getting in contact with the Simkl Team and we will figure something out. 

@ivarbrandt im very sorry for bringing you into this, from doing my early testing, I saw descriptions and tested other skins and thought maybe Fentastic could be having an issue with that. I apologize for that. Thank you for replying in here and trying to help us.

**Comment by ivarbrandt** [comment]
> im very sorry for bringing you into this

Don't be. All good man.

I tested a few changes on FENtastic without any success. If SIMKL lists come back without any metadata then there would be nothing for the skin to display. What I'm wondering is why the Arctic skins are showing metadata on SIMKL content. This could be something else though feature wise that jurial's skins have baked into them.

Anyways, let me know if you need anything else from me.

**Comment by jewbmx** [comment]
This is sorta off topic and highjacking the issue/chat, but since i seen the title as a watchlist account type i thought id ask here and not bug gujal with my question lol...
I was curious which watchlist option is the most worthy and possibly has its own phone app like trakt does :)  personally i been just using my trakt account with some manual updates for season issues and a combination of saved search results to keep the good shows that might continue on down the road lmao

**Comment by Goldenfreddy0703** [comment]
Hey there, I would most likely say that Simkl would be a great phone app sense it's similar to Trakt, 2nd I would say Anilist, 3rd I would say Mal, 4th I would say kitsu.

**Comment by Goldenfreddy0703** [comment]
Hey @guenguer , sorry it took forever but i finally managed to add metadata to simkl thanks to anilist. Hope this helps and im gonna go ahead and close this issue. If you have any issues relating to this issue, please feel free to reply and we will look further into this. 

Thanks
